### ISSUE TITLE
Randomly getting an empty <body> that gets cached

### ISSUE THREAD

**Issue by vlauria** [magento/magento2#9050]
## Preconditions

1. Magento CE  2.1.5 without sample data is installed.

## Steps to reproduce
I do not know.
I'm having problems with a very basic Magento 2 install, with very few customizations or extensions.
We randomly get a completely empty <body> tag on the home page that then gets cached on Varnish and the site is not accessible. All the <head> contents etc are just fine (except for an empty <title>).
Cleaning cache and banning Varnish fixes the issue but it's a huge issue.
I haven't found a way to replicate this as it seems to happened randomly.
If I load something like /?test that bypasses cache, the page loads correctly.

## Expected result

Homepage showed

## Actual result
Blank page

**Comment by davidalger** [comment]
@vlauria We've run into this issue on a couple EE builds, and the solution I believe was found in a patch which removed `layout="admin-1column"` from this file: https://github.com/magento/magento2/blob/develop/app/code/Magento/Ui/view/base/layout/default.xml

It's absolutely not an official answer, but we were told the following: _The target release where this fix will be included is 2.1.8 - however this is subject to change._ So you may want to use the following to patch your site in the mean time:

```
diff --git a/vendor/magento/module-ui/view/base/layout/default.xml b/vendor/magento/module-ui/view/base/layout/default.xml
index 7f2efbd..64d5f14 100644
--- a/vendor/magento/module-ui/view/base/layout/default.xml
+++ b/vendor/magento/module-ui/view/base/layout/default.xml
@@ -5,7 +5,7 @@
  * See COPYING.txt for license details.
  */
 -->
-<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" layout="admin-1column" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
     <body>
         <referenceContainer name="after.body.start">
             <block class="Magento\Ui\Block\Logger" name="logger" template="Magento_Ui::logger.phtml"/>
```

**Comment by miguelbalparda** [comment]
https://github.com/magento/magento2/pull/9560 seems to fix this issue, can you try that fix and report back? I'd like to merge that PR but I will need some help from you :smile:

**Comment by magento-team** [comment]
Internal ticket to track issue progress: MAGETWO-69018

**Comment by magento-team** [comment]
Internal ticket to track issue progress: MAGETWO-80647

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-nazar. Thank you for working on this issue. 
Looks like this issue is already verified and confirmed. But if your want to validate it one more time, please, go though the following instruction:
- [ ] 1. Add/Edit `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.

- [ ] 2. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 3. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 4. If the issue is not relevant or is not reproducible any more, feel free to close it.

**Comment by ghost** [comment]
Hi @vlauria The issue was re-tested and we can confirm that it was fixed on the 2.3 release branch. We closing this issue as fixed due to upcoming 2.3 release that will be available soon.
